### DISCUSSION TITLE
LLOneBot相关

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#456] [一般讨论]
正在迁移，毕竟这玩意也算CQHTTP/OneBot
想问一下这边对消息段/CQ码这个选项以及其他选项是不是有什么讲究

**Comment by Tsuk1ko** [comment]
应该是无所谓的，我看 LLOneBot 上报消息有 `raw_message` 字段，不管这个选什么都不影响程序拿到 cq 码形式的消息

以前专门做过兼容
